src/CMakeLists.txt

```diff
@@ -33,10 +33,13 @@ IF(ENABLE_TRACE)
 ENDIF()
 ADD_FEATURE_INFO(tracing GIT_TRACE "tracing support")
 
-CHECK_FUNCTION_EXISTS(futimens HAVE_FUTIMENS)
+enable_warnings(error)
```

Comment:
Is the `ENABLE_WARNINGS` / `DISABLE_WARNINGS` functional here?  Does `CHECK_SYMBOL_EXISTS` produce some warning in some cases that you're relying on failing the build?  Or is this leftover from debugging?  This isn't clear to me.

Reply:
"Check symbol exists" functions generates only a warning and "futimens" function treated as found.
"enable_warnings(error)" forces cmake to treat futimens properly ( not found ).

Test script
```sh
#!/bin/sh

export MACOSX_DEPLOYMENT_TARGET=10.10

rm -rf "xcode"
mkdir "xcode"
cd "xcode"
cmake -G "Xcode" ..
```

Reply:
I see, that makes sense.